recipe/meta.yaml

```diff
@@ -8,33 +8,32 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://github.com/UV-CDAT/libcdms/archive/v3.0.1.tar.gz
-  sha256: 9e62f77562f667db559466092cb0d908abf67a3216f7faaf152348a72f46786e
+  sha256: 8b53cd7821feefa0b357ca5127d9fc6164380e51f2205e7d2c5018f71a617f91
 
 build:
-  number: 0
+  number: 2
   skip: True  # [win] 
 
 requirements:
   build:
-    - hdf5 1.10.*
-    - libnetcdf 4.4.*
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+  host:
+    - libnetcdf >4.5
     - jasper
     - g2clib
     - libpng >=1.6.28,<1.7
     - libtiff 4.0.*
     - jpeg 9*
     - libdrs
-    - gcc
   run:
-    - hdf5 1.10.*
-    - libnetcdf 4.4.*
+    - libnetcdf >4.5
```

Comment:
This pinning is wrong. You are assuming that libnetcdf 4.6 is compatible with 4.7.
You should remove the version pinnings for `libnetcdf, libpng, jpeg, libtiff` and rerender. It will give `libnetcdf 4.6, libpng 1.6.34, libtiff 4.0.9, jpeg 9`